packages/rich-text-react-renderer/src/__test__/documents/embedded-entry.ts

```diff
@@ -0,0 +1,17 @@
+import { Document, BLOCKS } from '@contentful/rich-text-types';
+
+export default function(entry: Object) {
```

Comment:
I think we can make entry type more specific using an Entry type.
You can copy the type definition from the SDK [index.d.ts](https://github.com/contentful/contentful.js/blob/master/index.d.ts#L73-L77). WDYT?

Reply:
I tried to use the `Entry` type, however, it led to a bunch of issues: this isn't a full Contentful entry, as you can see [here](https://github.com/contentful/rich-text/pull/71/files#diff-c25141e0c89b52284671547a52d73c68R99). Considering that these are only used in the tests, I'd argue that we don't need to be that specific with the entry type.

Reply:
Good point about Entry type

```suggestion
export default function(entry: {sys: Contentful.Sys}) {
```

or even `Partial<Entry>` could work here.

Reply:
but I think we can take care of the mocks in one of the follow-ups

Reply:
Do you wanna tackle this when you [reorganise the mocks](https://github.com/contentful/rich-text/pull/71#discussion_r254430417)? These documents are copied one for one from the `rich-text-html-renderer`, including types. This should be updated there as well.